### PR TITLE
DataGrid - using toolbar options in demos

### PR DESCRIPTION
**Pull request by pomahtri**

### CODE REVIEW

File: JSDemos/Demos/DataGrid/CellEditingAndEditingAPI/Vue/App.vue [hunk 1]
```diff
@@ -17,7 +16,7 @@
       <DxPaging :enabled="false"/>
       <DxSelection mode="multiple"/>
       <DxColumn
-        :width="55"
+        width="55"
```
Comment (vconst) [inline JSDemos/Demos/DataGrid/CellEditingAndEditingAPI/Vue/App.vue]: Why?

File: JSDemos/Demos/DataGrid/MultipleRecordSelectionAPI/React/App.js [hunk 2]
```diff
@@ -61,6 +49,24 @@ class App extends React.Component {
           <Column dataField="Position" width={180} />
           <Column dataField="BirthDate" dataType="date" width={125} />
           <Column dataField="HireDate" dataType="date" width={125} />
+          <Toolbar>
+            <Item location="before">
+              <SelectBox
+                dataSource={['All', 'Dr.', 'Mr.', 'Mrs.', 'Ms.']}
```
Comment (vconst) [inline JSDemos/Demos/DataGrid/MultipleRecordSelectionAPI/React/App.js]: extract array to constant

File: JSDemos/Demos/DataGrid/MultipleRecordSelectionAPI/Angular/app/app.component.html [hunk 3]
```diff
@@ -29,6 +16,24 @@
     <dxi-column dataField="Position" [width]="180"></dxi-column>
     <dxi-column dataField="BirthDate" dataType="date" [width]="125"></dxi-column>
     <dxi-column dataField="HireDate" dataType="date" [width]="125"></dxi-column>
+    <dxo-toolbar>
+        <dxi-item location="before">
+            <dx-select-box
+                [dataSource]="['All', 'Dr.', 'Mr.', 'Mrs.', 'Ms.']"
```
Comment (vconst) [inline JSDemos/Demos/DataGrid/MultipleRecordSelectionAPI/Angular/app/app.component.html]: Initialize array as component variable

**Review by vconst — CHANGES_REQUESTED** [review]

File: JSDemos/Demos/DataGrid/CellEditingAndEditingAPI/Vue/App.vue (hunk 1 above)
Comment (pomahtri) [reply JSDemos/Demos/DataGrid/CellEditingAndEditingAPI/Vue/App.vue]: thought that it's best practise not to bind consts, [but it's not](https://ru.vuejs.org/v2/guide/components-props.html#%D0%9F%D0%B5%D1%80%D0%B5%D0%B4%D0%B0%D1%87%D0%B0-%D1%87%D0%B8%D1%81%D0%B5%D0%BB)

**Review by vconst — APPROVED** [review]

**Review by Alyar666 — APPROVED** [review]

**Pull request merged**